api/src/main/java/org/openmrs/ChemoDosingInstructions.java

```diff
@@ -27,6 +27,12 @@
 	private String dosageDeliveredTiming;
 
 	private String dosageDilutionInstructions;
+	
+	private static final String adjustmentPercentage = "Dosage Adjustment Percentage";
```

Comment:
Its a best practice to use capital letters when naming the constant variables. example [https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/util/OpenmrsConstants.java](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/util/OpenmrsConstants.java )

Reply:
Yeah, it skipped out of my mind. Thanks for pointing it out. Done the changes!

Reply:
helo you might look into squashing the commits into one

Reply:
Done